charts/horizon/values.yaml

```diff
@@ -118,7 +103,11 @@ ingest:
     ## If you'd like to have prometheus
     enabled: true
 
-    ## See global.image.coreExporter for image settings
+    image:
+      registry: docker.io
+      repository: stellar/stellar-core-prometheus-exporter
+      tag: latest
+      pullPolicy: Always
```

Comment:
We were planning on building full stellar network stacks and for that to work we needed the parent chart to be able to set the exporter. The exporter will also be used by the core chart which deploys validators.

Reply:
the parent chart can still set the exporter here